src/publish.proj

```diff
@@ -52,8 +52,7 @@
     <CreateAzureContainer Condition="'$(PublishFlatContainer)' == 'true'"
                           AccountKey="$(AccountKey)"
                           AccountName="$(AccountName)"
-                          ContainerName="$(ContainerName)"
-                          IsPublic="true" />
```

Comment:
Should we explicitly pass IsPublic=false?

Reply:
Probably, let me change that.